examples/combobox/combobox-autocomplete-both.html

```diff
@@ -54,7 +52,18 @@ <h2 id="ex_label">Example</h2>
           <div class="group">
             <input id="cb1-input" class="cb_edit" type="text" role="combobox" aria-autocomplete="both"
               aria-expanded="false" aria-controls="lb1">
-            <button type="button" id="cb1-button" aria-label="Open" tabindex="-1">&#9660;</button>
+            <button type="button" id="cb1-button" aria-label="Open" tabindex="-1">
+              <span class="arrow up">
+                <svg width="18" height="18">
```

Comment:
I think it's better to use a single SVG and instead use CSS `transform: rotate(180deg)` to change between up arrow and down arrow.

Reply:
I have removed one SVG image and used the transform property on the remaining image.